packages/consensus/src/validation/transaction.cairo

```diff
@@ -48,7 +53,7 @@ pub fn validate_transaction(
                 }
             }
 
-            if !is_input_final(*input.sequence) {
+            if *tx.version >= 2 && !is_input_final(*input.sequence) {
```

Comment:
#285 seems to be caused by the fact that tx version is set to 1, while sequence field seems to indicate timelock:
![image](https://github.com/user-attachments/assets/090d1698-a20f-4aa5-aac5-039c2c42e9c4)
@m-kus wdyt?

Reply:
I wonder what is the point of setting sequence to a non zero value if version < 2?